python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -151,7 +151,6 @@ cpdef read_parquet(filepaths_or_buffers, columns=None, row_groups=None,
 
     cdef bool cpp_strings_to_categorical = strings_to_categorical
     cdef bool cpp_use_pandas_metadata = use_pandas_metadata
-
```

Comment:
We could revert this file if changes are not necessary.

Reply:
Done